app/src/main/res/layout/row_item_pending_message.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  Copyright (c) 2016-2017 Onegini B.V.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  -->
+
+<android.support.v7.widget.CardView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_marginTop="@dimen/activity_margin"
+    android:layout_marginLeft="@dimen/activity_margin"
+    android:layout_marginRight="@dimen/activity_margin"
+    android:layout_height="wrap_content">
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:padding="8dp">
+
+        <TextView
+            android:id="@+id/pending_message_profile"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            tools:text="Robert"
```

Comment:
srsly?

Reply:
What's wrong with the name? Notice, that it's not `android:text`. `Tools` are stripped out from the app, it doesn't affect the app in runtime in any way. It's only used to make the layout preview look better when you use the IDE. It's like a code comment, as long as they are not stupid or offensive I'd keep them.

Reply:
I'm not saying you should remove them, I think You could change them so something like <user name goes here>

Reply:
IMO it should show an example of expected value, because you want to see the layout, how it looks like.

Reply:
It's  a valid point, ok.